specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2023-02-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,8540 @@
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/glossary/terms": {
+      "put": {
+        "tags": [
+          "Glossary"
+        ],
+        "description": "Update terms in bulk.",
+        "operationId": "Glossary_BulkUpdateTerms",
+        "x-ms-examples": {
+          "Glossary_BulkUpdateGlossaryTerms": {
+            "$ref": "./examples/Glossary_BulkUpdateGlossaryTerms.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "terms",
+            "description": "An array of terms to update.",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "description": "An array of glossary terms.",
+              "items": {
+                "$ref": "#/definitions/AtlasGlossaryTerm"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. An array of glossary terms.",
+            "schema": {
+              "type": "array",
+              "description": "An array of glossary terms.",
+              "items": {
+                "$ref": "#/definitions/AtlasGlossaryTerm"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Glossary"
+        ],
+        "description": "Get terms by guids.",
+        "operationId": "Glossary_ListTermsByGuids",
+        "x-ms-examples": {
+          "Glossary_ListGlossaryTermsByGuids": {
+            "$ref": "./examples/Glossary_ListGlossaryTermsByGuids.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "guid",
+            "in": "query",
+            "description": "An array of GUIDs of terms to list.",
+            "required": true,
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi",
+            "x-ms-client-name": "guids"
+          },
+          {
+            "$ref": "#/parameters/excludeRelationshipTypes"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. An array of glossary terms or an empty list.",
+            "schema": {
+              "type": "array",
+              "description": "An array of glossary terms or an empty list.",
+              "items": {
+                "$ref": "#/definitions/AtlasGlossaryTerm"
+              }
```

Comment:
Response should not be a bare array. The response should be an object with a top-level property that is an array.

If this operation is borrowed from open-source, then this is allowed. Otherwise it should be changed as stated.

Reply:
Thanks @mikekistler. This response pattern is also following the atlas behavior. 
![image](https://github.com/Azure/azure-rest-api-specs/assets/78060542/f47de200-8e12-4c57-b0ae-bde4d0a352a9)

Comments addressed. Please help review the changes. Thanks again.